src/core.ts

```diff
@@ -42,8 +42,8 @@ export class TemplateProcessor {
     return comitter.parts;
   }
 
-  handleTextExpression(node: Node, templateFactory: TemplateFactory) {
-    return new NodePart(node, node.nextSibling!, templateFactory);
+  handleTextExpression(templateFactory: TemplateFactory) {
```

Comment:
Must we remove the node argument here? Without that does this server any purpose? What can you do?

Reply:
Many we can leave it but not use it? This is just separating part creation from attachment.